### DISCUSSION TITLE
Typescript Support

### DISCUSSION THREAD

**Discussion by oroalej** [lupas/nuxt-fire-vuexfire-example#19] [Q&A]
Hi,

Does typescript supported in vuexFire?

**Comment by arnaldoperez** [comment]
I'm working on a a typescript project and I can't access this.$fire from my store folder on nuxt. So I had to create another firebase instance for my state, witch is not optimal but would do until someone answers this issue.... 

BTW: It would also be great if it could work with vuex-decorator-module or vuex-class-component...
